data/fields/crossing/whistle_yes.json

```diff
@@ -0,0 +1,20 @@
+{
+    "key": "crossing:whistle",
+    "type": "combo",
+    "label": "Train Horn",
+    "placeholder": "Mandatory",
+    "strings": {
+        "options": {
+            "optional": "Only as Needed",
+            "wayside": "Wayside/Stationary Horn"
+        }
+    },
+    "autoSuggestions": false,
+    "customValues": false,
+    "locationSet": {
+        "include": [
+            "ca",
+            "us"
+        ]
```

Comment:
[This study](https://www.sciencedirect.com/science/article/abs/pii/S0003687021000806) finds that Australia is extremely inconsistent in horn usage but notes that sounding the horn is the default.